src/main/java/dk/sdu/se_f22/sortingmodule/infrastructure/domain/SortingModuleImpl.java

```diff
@@ -68,12 +68,24 @@ public void setScoring(int scoring) {
 
     @Override
     public SearchHits search() {
+        // Save the query
+        this.saveSearch();
+
+        // Search
         SearchModuleImpl searchModule = new SearchModuleImpl();
```

Comment:
Not part of the changes, but the interface `SearchModule` should be used as the datatype